activerecord/lib/active_record/schema_dumper.rb

```diff
@@ -191,14 +191,14 @@ def indexes(table, stream)
             ]
             statement_parts << 'unique: true' if index.unique
 
-            index_lengths = (index.lengths || []).compact
-            statement_parts << "length: #{Hash[index.columns.zip(index.lengths)].inspect}" if index_lengths.any?
-
-            index_orders = index.orders || {}
-            statement_parts << "order: #{index.orders.inspect}" if index_orders.any?
+            statement_parts << "length: #{index.lengths.inspect}" if index.lengths.present?
```

Comment:
If you are guaranteed to have array here, it's preferable to use `.any?` over `.present?`